presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.base.security;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.prestosql.spi.connector.CatalogSchemaTableName;
+import io.prestosql.spi.security.AccessDeniedException;
+import io.prestosql.spi.security.Identity;
+import io.prestosql.spi.security.SystemAccessControl;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.security.auth.kerberos.KerberosPrincipal;
+
+import java.io.File;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.google.common.io.Files.copy;
+import static io.prestosql.plugin.base.security.FileBasedAccessControlConfig.SECURITY_CONFIG_FILE;
+import static io.prestosql.plugin.base.security.FileBasedAccessControlConfig.SECURITY_REFRESH_PERIOD;
+import static io.prestosql.spi.testing.InterfaceTestUtils.assertAllMethodsOverridden;
+import static java.lang.Thread.sleep;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.util.Files.newTemporaryFile;
+
+public class TestFileBasedSystemAccessControl
```

Comment:
In `Move SystemAccessControl implementations to presto-plugin-toolkit` commit -- why this file does not appear as moved?

What with `presto-main/src/test/java/io/prestosql/security/TestFileBasedSystemAccessControl.java`?

Reply:
It was copied and modified a bit. Original tests are testing things on different level (using `AccessControl`), here I am using `FileBasedSystemAccessControl` directly.